third_party/github.com/gjtorikian/isBinaryFile/BUILD.bazel

```diff
@@ -1,21 +1,19 @@
-load("@build_bazel_rules_nodejs//internal/js_library:js_library.bzl", "js_library")
-
 package(default_visibility = ["//visibility:public"])
 
 licenses(["notice"])
 
 exports_files(["LICENSE"])
 
-js_library(
-    name = "isBinaryFile",
+filegroup(
+    name = "package_contents",
     srcs = [
-        "src/index.js",
-        "package.json",
-    ],
-    package_name = "isbinaryfile",
+        "BUILD.bazel",
+        "LICENSE",
+        "//third_party/github.com/gjtorikian/isBinaryFile/src:package_contents",
+    ]
 )
 
-filegroup(
-    name = "package_contents",
-    srcs = glob(["*"]) + [":isBinaryFile"],
+alias(
```

Comment:
now that the ts_project rule is fixed I guess you can remove that nested BUILD file?

Reply:
👍